doc/sphinx/reference/onedim/nonlinear-solver.md

```diff
@@ -1,5 +1,387 @@
+  components are considered in the convergence check by providing a minimum threshold.
+
+#### Convergence Criterion
+
+The Newton iteration is considered converged when the weighted norm is less than 1:
+
+$$
+\sum_{n,j} \left(\frac{s_{n,j}}{w_n}\right)^2 < 1
+$$
+
+This criterion indicates that each component of the step vector $s_{n,j}$ is
+sufficiently small relative to the expected precision (as defined by the weights
+$w_n$).
+
+## Transient Solution
+
+There will be times when the solution of the steady-state problem can not be found
+using the damped Newton method. In this case, a transient solution is solved and a
+specified number of time steps are taken before the steady-state damped Newton
+method is attempted again.
+
+
+The equation that is being solved for the transient case is:
+
+$$
+\frac{d x}{dt} = F_{ss}(x)
+$$
+
+Where $F_{ss}(x)$ is the residual vector for the steady-state problem. That is, the
+residual vector that arises when the $\frac{d x}{dt}$ term is zero in the governing
+equations. The transient solution is solved using the implicit backward Euler method.
+The solution at the next time step is given by:
+
+$$
+\frac{x_{n+1} - x_n}{\Delta t} = F_{ss}(x_{n+1})
+$$
+
+Here the `n+1` is the solution at the next time step, `n` is the solution at the current
+time step.
+
+We consider a case where each element of the residual vector may not have a
+corresponding time derivative term. These equations without time derivative terms are
+referred to as algebraic equations, and the ones with time derivative terms are
+referred to as differential equations. A general way to express this is by writing the
+equation above in the following form.
+
+$$
+\alpha \frac{x_{n+1} - x_n}{\Delta t} + F_{ss}(x_{n+1})
+$$
+
+Where $\alpha$ is a diagonal matrix with diagonal values that are equal to 1 for
+differential equations and 0 for algebraic equations.
+
+Moving all terms to the right hand side of the equation, we get our expression for the
+residual equation that we will by solving:
+
+$$
+F(x_n, x_{n+1}) = -\frac{\alpha}{\Delta t}(x_{n+1} - x_n) + F_{ss}(x_{n+1})
+$$
+
+For the Newton method, we linearize the residual equation about the solution vector at
+the next iteration(not timestep) by using a Taylor series expansion. The linearized
+equation is given by:
+
+$$
+F(x_n, x_{n+1}) \approx F(x_n, x_{n+1}^{(k)}) +
+  \frac{\partial F(x_n, x_{n+1}^{(k)})}{\partial x_{n+1}} \Delta x_{n+1}^{(k)}
+$$
+
+Where $x_{n+1}^{(k)}$ is the solution at the $k$-th Newton iteration for the time step
+$n+1$. The Jacobian is the derivative term that is multiplying the correction vector
+$\Delta x_{n+1}^{(k)}$. The Jacobian is given by:
+
+$$
+J(x_n, x_{n+1}^{(k)}) = \frac{\partial F(x_n, x_{n+1}^{(k)})}{\partial x_{n+1}}
+$$
+
+Using the expression for the residual equation defined earlier, the Jacobian matrix can
+be written as:
+
+$$
+J(x_n, x_{n+1}^{(k)}) = \frac{\alpha}{\Delta t} I +
+  \frac{\partial F_{ss}(x_{n+1}^{(k)})}{\partial x_{n+1}}
+$$
```

Comment:
In the first term on the RHS, you could leave out $I$, since $\alpha$ is already defined as being a diagonal matrix. And it might help to write the second term as $J_{ss}(x_{n+1}^{(k)})$, which helps show why we're able to re-use the Jacobian between the steady and transient problems.